mobile/lib/auth/email_auth_widget.dart

```diff
@@ -66,7 +65,7 @@ class EmailAuthWidgetState<T extends EmailAuthWidget> extends State<T> {
   void clearEmailCallBack() {
     if (_emailAddress == '') {
       FocusScope.of(context).unfocus();
-      Future.delayed(const Duration(milliseconds: 250), () {
+      Future.delayed(const Duration(milliseconds: 400), () {
```

Comment:
increasing the time it takes to display signup options to prevent overflow issues after a user is done entering the email address